be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/Resources/AccountResources.en.resx

```diff
@@ -144,6 +144,9 @@
   <data name="EmailIsConfirmedText" xml:space="preserve">
     <value>Account Confirmed</value>
   </data>
+  <data name="EmailMarketingApprovalLabel" xml:space="preserve">
+    <value>I consent to receiving information from ELTAP electronically, to an e-mail address that I secure</value>
```

Comment:
**suggestion:** Word ELTAP couldn't be in text, because this app is not used only by ELTAP